src/index.js

```diff
@@ -1,177 +1,177 @@
-var processing = require("./processingHog"),
-    distance = require("ml-distance");
+'use strict';
+
+const processing = require('./processingHog');
 
 const PI_RAD = 180 / Math.PI;
 
 module.exports = {
-  extractHOG: extractHOG,
-  extractHistograms: extractHistograms
-}
+    extractHOG: extractHOG,
+    extractHistograms: extractHistograms
+};
 
 // also export all the functions from processing.js
 for (var func in processing) {
-  module.exports[func] = processing[func];
+    module.exports[func] = processing[func];
 }
 
 /**
  * Extract the HOG of an image
- * @param {image} Image - image to transform into a HOG descriptor
+ * @param {image} image - image to transform into a HOG descriptor
  * @param {object} options
  * @return {Array} Array with the value of the HOG descriptor
  */
 
 function extractHOG(image, options = {}) {
-  var blockSize = options.blockSize || 2;
-  var blockStride = options.blockStride || (blockSize / 2);
-  var norm = options.norm || "L2";
-
-  var histograms = extractHistograms(image, options);
-
-  var blocks = [];
-  var blocksHigh = histograms.length - blockSize + 1;
-  var blocksWide = histograms[0].length - blockSize + 1;
-
-  for (var y = 0; y < blocksHigh; y += blockStride) {
-    for (var x = 0; x < blocksWide; x += blockStride) {
-      var block = getBlock(histograms, x, y, blockSize);
-      normalize(block, norm);
-      blocks.push(block);
+    const {
+        blockSize = 2,
+        norm = 'L2'
+    } = options;
+    const blockStride = options.blockStride || (blockSize / 2);
+
+    var histograms = extractHistograms(image, options);
+
+    var blocks = [];
+    var blocksHigh = histograms.length - blockSize + 1;
+    var blocksWide = histograms[0].length - blockSize + 1;
+
+    for (var y = 0; y < blocksHigh; y += blockStride) {
+        for (var x = 0; x < blocksWide; x += blockStride) {
+            var block = getBlock(histograms, x, y, blockSize);
+            normalize(block, norm);
+            blocks.push(block);
+        }
     }
-  }
-  return Array.prototype.concat.apply([], blocks);
+    return Array.prototype.concat.apply([], blocks);
```

Comment:
not the good shape, it doesn't work without Array.prototype.concat.apply([], blocks).